services/QuillLMS/client/app/bundles/Teacher/components/premium/infoTooltip.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+
+import { Tooltip, } from '../../../Shared/index'
+
+const helpIconSrc = `${process.env.CDN_URL}/images/icons/icons-help.svg`
```

Comment:
I'm wondering if we should maybe just pull all these icon srcs we have throughout the codebase into the shared library at some point... what are your thoughts?

Reply:
I think that's a really good idea. I'll start that in the teacher dashboard project.